docs/workers.md

```diff
@@ -35,26 +55,26 @@ the correct worker, or to the main synapse instance. Note that this includes
 requests made to the federation port. See [reverse_proxy.md](reverse_proxy.md)
 for information on setting up a reverse proxy.
 
-To enable workers, you need to add *two* replication listeners to the
-main Synapse configuration file (`homeserver.yaml`). For example:
+To enable workers, you need to add both a HTTP replication listener and redis
+config to the main Synapse configuration file (`homeserver.yaml`). For example:
```

Comment:
should we describe `homeserver.yaml` as a "shared configuration file" or something?

Reply:
Good point, now that we're using "main" to mean the main synapse process.